### DISCUSSION TITLE
Is there a way to combine Rubin data with data from other surveys?

### DISCUSSION THREAD

**Discussion by bnord** [lsst-epo/citizen-science-notebooks#58] [Q&A]
1. What if people want to mix data from another survey? 
    1. E.g., Rubin data AND data that comes from SDSS from the same part of the sky?
    2. . What about simulated data to help with testing

**Comment by ericdrosas87** [comment]
@bnord Are you thinking Rubin data and non-Rubin data within the same Zooniverse subject set? Different subject sets within the same workflow? Different workflows entirely? All of the above?

**Reply by bnord** [reply]
INitially, I was thinking of Rubin and non-Rubin in the same subject set. I can imagine use cases for that.

Perhaps there are use cases for the other scenarios you mentioned, but I haven't yet thought of examples.

**Reply by ericdrosas87** [reply]
Okay and within the same subject set are you thinking non-Rubin image data? Or are you thinking Rubin image data and non-Rubin tabular data? Or both?

Edit: Changed `non-Rubin image data` to `non-Rubin tabular data` in the second question

**Reply by bnord** [reply]
to start, I'm thinking image data for both.

**Reply by ericdrosas87** [reply]
Technically it's possible right now to send image data from non-Rubin data sources. For example, I could use the `Citizen_Science_Testing` notebook to curate 5 images and automatically generate the `manifest.csv`. Then manually upload 5 images from some other image survey to the same `/cutouts` folder in the Notebook fileystem, and manually edit the `manifest.csv` to reference these 5 additional images. And then running the `send_image_data()` cell would send both data as one subject set.

It's also possible as I'm sure you know to annotate a column in the `manifest.csv` using the `metadata:` prefix to differentiate data sources on the Zooniverse project/workflow pages so citizen scientists will know which data came from where - if that's important/desirable.

So what's missing, I think, is a notebook that specifically queries some other non-Rubin data source _and_ the RSP databases to combine everything into one subject set automatically using the same or slightly modified PyPI package/`utils.py` (without manual upload, manual editing of the `manifest.csv`, etc.).

Let me know if you would like clarification on any of that or have any questions!

**Reply by bnord** [reply]
Thanks for explaining this. This is exactly what I was trying to understand now for our planning. I'll re-open this inquiry if/when we know how to pursue this. tx.
